src/app/core/entity-components/entity-details/entity-details.component.ts

```diff
@@ -142,6 +142,7 @@ export class EntityDetailsComponent {
           { duration: 8000 }
         );
         snackBarRef.onAction().subscribe(() => {
+          this.entity._rev = undefined;
           this.entityMapperService.save(this.entity, true);
```

Comment:
We are using the `forceOverwrite` flag. In case the saving of the entity fails (like in this case due to the wrong rev), a conflict resolution is started which loads the existing entity and tries to solve the conflict. In case for a deleted document I think the function throws an error because the `await this.get(newObject._id)` fails ([see here](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/database/pouch-database.ts#L192)). So I think you could add your solution to that function to make it more generally available.